patches/rollup-plugin-lib-style+1.2.10.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/rollup-plugin-lib-style/lib/index.js b/node_modules/rollup-plugin-lib-style/lib/index.js
+index 6a3823c..a055bb6 100644
+--- a/node_modules/rollup-plugin-lib-style/lib/index.js
++++ b/node_modules/rollup-plugin-lib-style/lib/index.js
+@@ -96,7 +96,7 @@ const defaultLoaders = [
+   },
+ ];
+ 
+-const replaceMagicPath = (fileContent) => fileContent.replace(MAGIC_PATH_REGEX, ".");
++const replaceMagicPath = (fileContent) => fileContent.replace(MAGIC_PATH_REGEX, "..");
```

Comment:
realised the plugin is a bit buggy when nested, so patching the package temporarily. 

will replace this with a fork of the plugin